src/objects.py

```diff
@@ -174,9 +174,13 @@ def get_actual_str(self):
     def has_valid_cost(self):
         return self.acc.has_valid_cost(self)
 
+    # return False on success
+    def fixup_invalid_cost(self):
+        return self.acc.fixup_invalid_cost(self)
```

Comment:
There is no method fixup_invalid_cost in PlanNodeAccessor:

```
class PlanNode:
    def __init__(self, accessor: PlanNodeAccessor, node_type):
        self.acc = accessor
```

Reply:
Added type hint to `self.acc`, too. The checker I installed ([flake8](https://flake8.pycqa.org/)) doesn't complain though.
```
class PlanNode:
    def __init__(self, accessor: PlanNodeAccessor, node_type):
        self.acc: PlanNodeAccessor = accessor
```